general/contact.rst

```diff
@@ -62,3 +60,5 @@ The core team consists of engineers from TrustedFirmware.org. Related, see the
 .. _core team: https://github.com/orgs/OP-TEE/teams/linaro/members
 .. _issues: https://help.github.com/articles/about-issues/
 .. _issues at optee_os: https://github.com/OP-TEE/optee_os/issues
+.. _Mailing Aliases: https://developer.trustedfirmware.org/w/collaboration/security_incident_process/mailing_aliases
```

Comment:
Sorry, I've moved this as well to https://developer.trustedfirmware.org/w/collaboration/security_center/mailing_aliases/. I thought the sub-pages would move when I moved the top level page but they didn't (sigh).

Reply:
Thanks Dan, I've updated it to the correct URL.